packages/rancher-alerting-drivers/charts/templates/cluster-role.yaml

```
@@ -0,0 +1,66 @@
+{{- $admin := "alerting-drivers-admin" -}}
+{{- $edit  := "alerting-drivers-edit"  -}}
+{{- $view  := "alerting-drivers-view"  -}}
+
+{{- if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" $admin ) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ $admin }}
+  labels:
+    {{- include "rancher-alerting-drivers.labels" . | nindent 4 }}
```

```suggestion
  labels: {{ include "rancher-alerting-drivers.labels" . | nindent 4 }}
```

For any nindent, place them on the same line and remove the preceding `-`